api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/services/status/APIDocRetrievalService.java

```diff
@@ -67,74 +57,41 @@ public APIDocRetrievalService(RestTemplate restTemplate,
      * @return the api docs as a string
      */
     public ResponseEntity<String> retrieveApiDoc(@NonNull String serviceId, String apiVersion) {
-        log.info("Attempting to retrieve API doc for service {} version {}",  serviceId, apiVersion);
```

Comment:
There is a lot of reverted code for static APIs in the API catalog.

Reply:
I don't understand can you be more specific ?
This branch was created off the current Zowe master.

Reply:
Working now. 

But I need to make some text changes to the new swagger before merging.
Will discuss tomorrow with Denis/Andrew.

![image](https://user-images.githubusercontent.com/20421943/47754397-06fbb180-dc9b-11e8-9302-30df90f83651.png)